packages/raylib/raylib.0.3.1/opam

```diff
@@ -0,0 +1,49 @@
+opam-version: "2.0"
+synopsis: "OCaml bindings for raylib"
+description: "OCaml bindings for raylib"
+maintainer: ["tobiasjammer@gmail.com"]
+authors: ["Tobias Mock"]
+license: "MIT"
+homepage: "https://github.com/tjammer/raylib-ocaml"
+bug-reports: "https://github.com/tjammer/raylib-ocaml/issues"
+depends: [
+  "dune" {>= "2.8"}
+  "dune-configurator"
+  "ctypes" {>= "0.14"}
+  "ppx_cstubs" {>= "0.6"}
+  "conf-mesa" {os = "linux" | os = "freebsd"}
+  "conf-libxcursor" {os = "linux" | os = "freebsd"}
+  "conf-libxi" {os = "linux" | os = "freebsd"}
+  "conf-libxinerama" {os = "linux" | os = "freebsd"}
+  "conf-libxrandr" {os = "linux" | os = "freebsd"}
```

Comment:
```suggestion
  "conf-mesa" {build}
  "conf-libxcursor" {build}
  "conf-libxi" {build}
  "conf-libxinerama" {build}
  "conf-libxrandr" {build}
```
are the `{os = "linux" | os = "freebsd"}` here for a particular reason? Are they really not needed on other OSs? Support for other OSs could you be added in the future in those packages.

Reply:
Yes, they are. The lib builds fine on other OS without depext, eg windows and macos https://github.com/tjammer/raylib-ocaml/actions/runs/837001056. But on these OS other (native) libs are linked instead of the X libs which are typical for linux. I don't want to stop people from building on other systems, where I don't know the exact dependencies.

I thought about using optional dependencies, but that doesn't really fit either. On linux/bsd these conf-packages are needed, and depexts facilitate easy installing. And for other systems other conf-packages might be needed.

Reply:
Oh ok I see, i understand. In which case, could you do:
```suggestion
  "conf-mesa" {os = "linux" | os-family = "bsd"}
  "conf-libxcursor" {os = "linux" | os-family = "bsd"}
  "conf-libxi" {os = "linux" | os-family = "bsd"}
  "conf-libxinerama" {os = "linux" | os-family = "bsd"}
  "conf-libxrandr" {os = "linux" | os-family = "bsd"}
```
instead, so that other BSDs can be compatible?